docs/pages/install.mdx

```diff
@@ -48,3 +39,44 @@ For example: 7.1.2
 - 2 represents a bug fix/patch release
 
 The download section above always shows the latest released version of Pelican and we recommend you to use the latest feature or major release version so it includes the latest bug fixes and features. If you would like to download a previous version, please refer to our [GitHub release page](https://github.com/PelicanPlatform/pelican/releases).
```

Comment:
Is this statement still true with the newer version of the Downloads table?

Reply:
I'll update this expression here